lib/vdsm/storage/managedvolume.py

```diff
@@ -108,10 +113,10 @@ def attach_volume(sd_id, vol_id, connection_info):
                     multipath_id=attachment.get("multipath_id"))
                 _invalidate_lvm_devices(attachment)
                 volume_type = connection_info["driver_volume_type"]
-                if volume_type not in ("rbd", "iscsi"):
+                if volume_type not in supported_drivers:
                     raise se.UnsupportedOperation(
-                        "Unsupported volume type, supported types are: "
-                        "rbd, iscsi")
+                        f"Unsupported volume type, supported types are: "
+                        "{supported_drivers}")
```

Comment:
Does this work? I'd say the second line of the string needs the `f` to be an f-string. The first line does not need it, but I think looks better if we keep it for consistency.

Reply:
yeah, fixed it locally